data/streams.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs');
+const concat = require('concat');
+const csv = require('csvtojson'); 
+const program = require('commander');
+
+function reverse(str) {
+    console.log(str.split("").reverse().join(""));
+}
+
+function transform(str) {
+    console.log(str.toUpperCase());
+}
+
+function outputFile(filePath) {
+    const reader = fs.createReadStream(filePath, {encoding: 'utf-8'});
+    reader.on('readable', () => {
+        console.log(reader.read());
+    });
+}
+
+function convertFromFile(filePath) {
+    csv().fromFile(filePath).then((jsonObj) => {
```

Comment:
Consider creating a read stream and using third party library just for transformation.
E.x. pipe the read stream to csv transformation logic and then to stdout.